drivers/hmis/app/graphql/types/hmis_schema/service.rb

```diff
@@ -8,6 +8,15 @@
 
 module Types
   class HmisSchema::Service < Types::BaseObject
+    # FIXME: order dependent, filter options must be first statement in class
```

Comment:
@gigxz I started encountering an error running these tests. Have you ever seen this kind of thing?
> "Variable $filters of type ServicesForEnrollmentFilterOptions! was provided invalid value for serviceType (Field is not defined on ServicesForEnrollmentFilterOptions)"

Seems like BaseFilterOptions is getting built before the available_filter_options method is defined. Probably just a local development issue but I thought it was odd that it is only manifesting now.

Reply:
Weird, I've never seen that before. I can reproduce it locally with test `rspec ./drivers/hmis/spec/requests/hmis/enrollment_service_filter_spec.rb` if I move the filter declaration down to where it used to be. I have no idea why it would be coming up now

Reply:
Not sure if this should be a `FIXME` comment, though, maybe just a comment